cmd/clusterctl/cmd/move.go

```diff
@@ -46,6 +49,14 @@ var moveCmd = &cobra.Command{
 		Move Cluster API objects and all dependencies between management clusters.
 		clusterctl move --to-kubeconfig=target-kubeconfig.yaml`),
 	Args: cobra.NoArgs,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		toFolder, _ := cmd.Flags().GetBool("to-folder")
+		fromFolder, _ := cmd.Flags().GetBool("from-folder")
+		if toFolder || fromFolder {
```

Comment:
```suggestion
		if mo.backupToFolder || mo.restoreFromFolder {
```
Also, would it be possible to check if `mo.directory` is emtpy instead of marking flag as mandatory? Looks like they effectively do the same thing but marking the flag as maandatory if something else if provided seems like an indirect way to do it.

Reply:
Out of curiosity: I remember that you initially had an implementation that used `MarkFlagsRequiredTogether` function. What was the problem with that?

Reply:
Good point! Fixed.

The problem with `MarkFlagsRequiredTogether` as I used it, was that if you set `directory` you also had to set from AND to folder. So it did exactly what it said it would do, but not what I wished it would do. :)